pkgs/development/libraries/structuresynth/default.nix

```diff
@@ -0,0 +1,54 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  qtbase,
+  wrapQtAppsHook,
+  libGL,
+  libGLU,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "structuresynth";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "alemuntoni";
+    repo = "structuresynth";
+    rev = finalAttrs.version;
+    hash = "sha256-uFz4WPwA586B/5p+DUJ/W8KzbHLBhLIwP6mySZJ1vPY=";
+  };
+
+  outputs = [
+    "dev"
+    "out"
+  ];
+
+  buildInputs = [
+    libGL
+    libGLU
+    qtbase
+  ];
+
+  nativeBuildInputs = [
+    cmake
+    wrapQtAppsHook
+  ];
+
+  postPatch = ''
+    substituteInPlace CMakeLists.txt \
+      --replace-warn STATIC SHARED \
+      --replace-warn '"'$'{CMAKE_CURRENT_LIST_DIR}"' \
+      '$<BUILD_INTERFACE:''${CMAKE_CURRENT_LIST_DIR}> $<INSTALL_INTERFACE:''${CMAKE_INSTALL_INCLUDEDIR}>'
+    echo "install(TARGETS structure-synth)" >> CMakeLists.txt
+    echo 'install(FILES ''${HEADERS} TYPE INCLUDE)' >> CMakeLists.txt
```

Comment:
Not a blocker for merging but please open a PR upstream. It would also be nice to generate a `*Config.cmake` file

Reply:
Yes, will do, thanks for the reminder

Reply:
done and merged: https://github.com/alemuntoni/StructureSynth/pull/1